torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <torch/types.h>
+
+namespace torch {
+namespace nn{
+namespace functional {
+
+inline Tensor bilinear(Tensor& input1, Tensor& input2, Tensor& weight, Tensor& bias) {
```

Comment:
I believe we would need to change the function signature to:
```suggestion
inline Tensor bilinear(const Tensor& input1, const Tensor& input2, const Tensor& weight, const Tensor& bias=Tensor()) {
```
because we don't change those input tensors in the function body 👯‍♂

Reply:
It would be awesome to add a test case for `auto y = F::bilinear(input1, input2, weight);` (i.e. no bias) use case as well. Thanks so much for your help @MJ10!